Dockerfile

```diff
@@ -17,13 +17,14 @@ ENV SASLAUTHD_MECH_OPTIONS=""
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Packages
-# hadolint ignore=DL3015
+# hadolint ignore=DL3015,SC2016
```

Comment:
I want to later look why the next exception is needed, but this should not block